ui/src/containers/NodeVolumes.js

```diff
@@ -77,6 +153,64 @@ const NodeVolumes = props => {
           />
         </span>
       )
+    },
+    {
+      label: intl.messages.action,
+      dataKey: 'action',
+      disableSort: true,
+      width: 80,
+      renderer: (data, rowData) => {
+        const isEnableClick = isVolumeDeletable(rowData);
+        const CustI = styled.i`
+          cursor: ${props => {
+            if (props.isEnableClick) {
+              return `pointer`;
+            } else {
+              return `not-allowed`;
+            }
+          }};
+        `;
+
+        const hintPopup = () => {
+          let hintMessage = '';
+
+          switch (rowData.status) {
+            case STATUS_PENDING:
+              hintMessage = intl.messages.volume_status_pending_hint;
+              break;
+            case STATUS_TERMINATING:
+              hintMessage = intl.messages.volume_status_terminating_hint;
+              break;
+            case STATUS_VOLUME_UNKNOWN:
+              hintMessage = intl.messages.volume_status_unknown_hint;
+              break;
+            case STATUS_FAILED:
```

Comment:
I'm not sure the string is good here: a `Failed` volume is not busy (and if the `PV` is not bound you can delete it)

If the volume is `Available` or `Failed` you need to check the `PV` to know what happen:
- PV is `Bound` => we should display a message saying `Volume is bound, cannot delete`
- PV is `Pending` => here is the `busy` case I guess (shouldn't happen so don't have a better message for this case).